src/main/java/Parser.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Parser {
+
+	private static final int MAX_WORDS_PER_COMMAND = 5;
+	private static final int START_OF_ITEMS = 1;
+	private static final int ACTION_WORD_POSITION = 0;
+
+	//Returns a String array where first element is action.
+	protected String getLine() {
+		Scanner in = new Scanner(System.in);
+		return in.nextLine();
+	}
+
+	private String[] getLineArray() {
+		Scanner parseScanner = new Scanner(System.in);
+		String line = parseScanner.nextLine();
+		line = line.toLowerCase();
+		Scanner lineScanner = new Scanner(line);
+		String[] command = new String[MAX_WORDS_PER_COMMAND];
```

Comment:
My understanding is that the remainder of the input more than 4 words is ignored? If this is the case, the system should throw some sort of exception to notify the user, otherwise they may not realize a part of their command has been ignored.